### PR TITLE
Fixed typo and removed unused vars

### PR DESCRIPTION
**Pull request by jrbasso**
Refs #408

### CODE REVIEW

File: github/timestamp_test.go [hunk 1]
```diff
@@ -92,7 +92,7 @@ func TestTimstamp_MarshalReflexivity(t *testing.T) {
 			t.Errorf("%s: Marshal err=%v", tc.desc, err)
 		}
 		var got Timestamp
-		err = json.Unmarshal(data, &got)
+		json.Unmarshal(data, &got)
```
Comment (willnorris) [inline github/timestamp_test.go]: rather than removing the unused vars, how bout we actually use them by calling t.Errorf?

Comment (jrbasso) [inline github/timestamp_test.go]: Indeed and fixed.

File: github/timestamp_test.go [hunk 2]
```diff
@@ -93,6 +93,9 @@ func TestTimstamp_MarshalReflexivity(t *testing.T) {
 		}
 		var got Timestamp
 		err = json.Unmarshal(data, &got)
+		if err != nil {
+			t.Errof("%s: Unmarshal err=%v", tc.desc, err)
```
Comment (shurcooL) [inline github/timestamp_test.go]: Did you try running this? It looks like this would be a build error. `Errof` should be `Errorf`.

**Comment by shurcooL** [comment]
LGTM, thanks!

**Comment by gmlewis** [comment]
LGTM.  Merging.

**Comment by gmlewis** [comment]
Forgot to add "Closes" line to commit, sorry.  Closing issue manually.

**Pull request merged**